cirq-google/cirq_google/calibration/workflow.py

```diff
@@ -268,13 +270,33 @@ def prepare_characterization_for_moments(
         operations matched by gates_translator, or it mixes a single qubit and two qubit gates.
     """
     if initial is None:
-        allocations: List[Optional[int]] = []
-        calibrations: List[RequestT] = []
-        pairs_map: Dict[Tuple[Tuple[Qid, Qid], ...], int] = {}
+        initial = []
+
+    if isinstance(circuit, Circuit):
```

Comment:
For clarity, I would create a list called `circuits` and just iterate over that: 

```python
circuits = [circuit] if isinstance(circuit, Circuit) else circuit 
```

Reply:
There is a different return type depending on each case, I don't think that I can unify those?